signal2html/core.py

```diff
@@ -125,7 +126,7 @@ def add_mms_attachments(db, mms, backup_dir, thread_dir):
 
 
 def get_mms_records(
-    db, thread, addressbook, backup_dir, thread_dir, version=None
+    db, thread, addressbook, backup_dir, thread_dir, versioninfo=None
```

Comment:
same with the ``versionInfo`` argument here

Reply:
Well, yes, but it is already needed in #19 (https://github.com/GjjvdBurg/signal2html/pull/19/files#diff-3ce0fcbfb4a5a5cfc47df0642f6e5a035f922740b74d9d3e438319223f1aebc3R173).

Reply:
Ah right, makes sense